src/custom/docs/components/navmenu/NavigationMenu.tsx

```diff
@@ -23,15 +25,37 @@ export const Navigation = (props: {
         { props.navTextLinks.map((item) => {
           return (
             <NavigationMenuItem key={item.title}>
-              <NavigationMenuLink
-                className={navigationMenuTriggerStyle()}
-                asChild
-                active={isActivePath(props.activePath, item.href)}
-              >
-                <props.linkComponent href={item.href}>
-                  {item.title}
-                </props.linkComponent>
-              </NavigationMenuLink>
+              <NavigationMenuTrigger>{item.title}</NavigationMenuTrigger>
+                <NavigationMenuContent>
+                  <ul className="grid gap-3 p-4 md:w-[400px] lg:w-[500px] lg:grid-cols-[.75fr_1fr]"> 
+                  <li className="row-span-4">
+                    <NavigationMenuLink asChild>
+                      <RegularLink
+                        className="flex h-full w-full select-none flex-col justify-end rounded-md bg-gradient-to-b from-muted/50 to-muted p-6 no-underline outline-none focus:shadow-md"
+                        href={item.href}
+                      >
+                        {item.logo}
+                        <p className="text-sm leading-tight text-muted-foreground py-6">
+                          {item.description}
+                        </p>
+                      </RegularLink>
```

Comment:
```suggestion
                      <RegularLink
                        className="flex h-full w-full select-none flex-col justify-end rounded-md bg-gradient-to-b from-muted/50 to-muted p-6 no-underline outline-none focus:shadow-md"
                        href={item.href}
                      >
                        {item.logo}
                        <p className="text-sm leading-tight text-muted-foreground py-6">
                          {item.description}
                        </p>
                      </RegularLink>
```
If regularLink is an a tag, I would just use an a tag instead

Reply:
done

Reply:
https://github.com/CQCL/quantinuum-ui/pull/34/commits/5ed49a3245968541b606884a33d1fa041e0b2a42

Reply:
https://github.com/CQCL/quantinuum-ui/commit/05de846bd0797896edb8ca5328f51abccebb4d9e